example_api/main.cpp

```diff
@@ -80,11 +80,11 @@ int main()
 	DumpHex(ttdcursor.GetThreadInfo(), 0x20);
 
 	std::cout << "\nContext:\n";
-	TTD::TTD_Replay_RegisterContext* ctxt = ttdcursor.GetCrossPlatformContext();
+	auto ctxt = ttdcursor.GetContextx86();
 	DumpHex(ctxt, 0xA70);
-	printf("RCX: %llx\n", ctxt->rcx);
+	printf("RCX: %llx\n", ctxt->Ecx);
```

Comment:
As a result, can we keep the x64 example?
The x86 could be highlighted (in a future PR) by docs or another example